tasmota/xsns_02_analog.ino

```diff
@@ -334,6 +370,32 @@ uint16_t AdcGetLux(uint32_t idx) {
   return (uint16_t)ldrLux;
 }
 
+float AdcGetMq(uint32_t idx) {
+  float avg = 0.0;
+  int retries = 5;
+  int retries_delay = 20;
+  float _RL = 10; //Value in KiloOhms
+  float _R0 = 10;
+  int _adc = AdcRead(Adc[idx].pin, 2);
+  avg += _adc;
+  for (int i = 0; i < retries-1; i ++) {
+    delay(retries_delay);
+    _adc = AdcRead(Adc[idx].pin, 2);
+    avg += _adc;
+  }
+  float voltage = (avg/ retries) * ANALOG_V33 / ((pow(2, ANALOG_RESOLUTION)) - 1);
```

Comment:
We are rtying to avoid using the heavy `pow(..)`standard function.
Please use Tasmota's `FastPrecisePow(...)` instead

Reply:
You right, sorry!
DONE